src/Runner/DefaultTestResultCache.php

```diff
@@ -178,17 +180,24 @@ public function serialize(): string
         ]);
     }
 
+    /**
+     * @param string $serialized
+     */
     public function unserialize($serialized): void
     {
         $data = \unserialize($serialized);
 
         if (isset($data['times'])) {
+            /** @var string $testName */
```

Comment:
I would prefer `\assert(\is_string($testName);`.

Reply:
Done. I'll repush in a second.